gitea/issue.go

```diff
@@ -114,12 +118,13 @@ func (c *Client) CreateIssue(owner, repo string, opt CreateIssueOption) (*Issue,
 
 // EditIssueOption options for editing an issue
 type EditIssueOption struct {
-	Title     string   `json:"title"`
-	Body      *string  `json:"body"`
-	Assignee  *string  `json:"assignee"`
-	Assignees []string `json:"assignees"`
-	Milestone *int64   `json:"milestone"`
-	State     *string  `json:"state"`
+	Title     string     `json:"title"`
+	Body      *string    `json:"body"`
+	Assignee  *string    `json:"assignee"`
+	Assignees []string   `json:"assignees"`
+	Milestone *int64     `json:"milestone"`
+	State     *string    `json:"state"`
+	Deadline  *time.Time `json:"due_date"`
```

Comment:
`deadline` should also be added to CreateIssueOption

Reply:
Add `// swagger:strfmt date-time`

Reply:
> deadline should also be added to CreateIssueOption

It is, see line 99.